### PR TITLE
Fabdouglas patch 1

### PR DESCRIPTION
**Pull request by kellyrob99**
Corrected version of #19

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -32,14 +32,18 @@ RUN mkdir -p /opt/sonatype/nexus \
...
 RUN sed \
     -e "s|karaf.home=.|karaf.home=/opt/sonatype/nexus|g" \
     -e "s|karaf.base=.|karaf.base=/opt/sonatype/nexus|g" \
     -e "s|karaf.etc=etc|karaf.etc=/opt/sonatype/nexus/etc|g" \
     -e "s|java.util.logging.config.file=etc|java.util.logging.config.file=/opt/sonatype/nexus/etc|g" \
     -e "s|karaf.data=data|karaf.data=${NEXUS_DATA}|g" \
     -e "s|java.io.tmpdir=data/tmp|java.io.tmpdir=${NEXUS_DATA}/tmp|g" \
-    -i /opt/sonatype/nexus/bin/nexus.vmoptions
+    -i /opt/sonatype/nexus/bin/nexus.vmoptions \
+  && sed \
```
Comment (DarthHater) [inline Dockerfile]: I'm mildly ignorant, but do you need another sed being it's already being used?

Comment (kellyrob99) [inline Dockerfile]: different file, different command

Comment (DarthHater) [inline Dockerfile]: Fair enough

**Comment by bradbeck** [comment]
An example usage of `NEXUS_CONTEXT` in `README.md` would be helpful.

**Comment by DarthHater** [comment]
@bradbeck added documentation

**Pull request merged**